architecture/additional_concepts/authentication.adoc

```diff
@@ -685,3 +685,186 @@ responses from `/oauth/authorize`, and how they should be handled:
+- apiVersion: v1
+  clientName: system:serviceaccount:myproject:jenkins
+  kind: OAuthClientAuthorization
+  metadata:
+    creationTimestamp: 2016-11-22T16:02:44Z
+    name: developer1::system:serviceaccount:myproject:jenkins
+    resourceVersion: "599"
+    selfLink: /oapi/v1/oauthclientauthorizations/developer1::system:serviceaccount:myproject:jenkins
+    uid: 1afc194d-b0cd-11e6-b245-507b9dac97ff
+  scopes:
+  - user:info
+  userName: developer1
+  userUID: fe3f3a4e-b0ca-11e6-ac48-507b9dac97ff
+- apiVersion: v1
+  clientName: system:serviceaccount:myproject:jenkins
+  kind: OAuthClientAuthorization
+  metadata:
+    creationTimestamp: 2016-11-22T15:48:39Z
+    name: developer1:system:serviceaccount:myproject:jenkins
+    resourceVersion: "508"
+    selfLink: /oapi/v1/oauthclientauthorizations/developer1:system:serviceaccount:myproject:jenkins
+    uid: 235cbdff-b0cb-11e6-ac48-507b9dac97ff
+  scopes:
+  - user:info
+  userName: developer1
+  userUID: fe3f3a4e-b0ca-11e6-ac48-507b9dac97ff
+- apiVersion: v1
+  clientName: system:serviceaccount:myproject:jenkins
+  kind: OAuthClientAuthorization
+  metadata:
+    creationTimestamp: 2016-11-22T15:47:12Z
+    name: developer:system:serviceaccount:myproject:jenkins
+    resourceVersion: "494"
+    selfLink: /oapi/v1/oauthclientauthorizations/developer:system:serviceaccount:myproject:jenkins
+    uid: ef5c24f2-b0ca-11e6-ac48-507b9dac97ff
+  scopes:
+  - user:info
+  userName: developer
+  userUID: cf29b7cb-b0ca-11e6-ac48-507b9dac97ff
+kind: List
+metadata: {}
+----
+<1> Each OAuth Client Authorization object is listed, starting with `apiVersion`.
+<2> Metadata added to identify the object.
+<3> The scope of the access request.
+
+As a non-administrator user, you can view the contents of your own OAuth Client Authorization object files.
+
+----
+# oc get selfoauthclientauthorization <file-client-name> -o yaml
+----
+
+For example:
+
+----
+# oc get selfoauthclientauthorization system:serviceaccount:myproject:jenkins -o yaml
+----
+
+The output is similar to the following:
+
+.Sample OAuth Client Authorization object file for a user
+----
+apiVersion: v1
+clientName: system:serviceaccount:myproject:jenkins
+kind: SelfOAuthClientAuthorization
+metadata:
+  creationTimestamp: 2016-11-22T16:02:44Z
+  name: system:serviceaccount:myproject:jenkins
+  resourceVersion: "599"
+  selfLink: /oapi/v1/selfoauthclientauthorizations/system:serviceaccount:myproject:jenkins
+  uid: 1afc194d-b0cd-11e6-b245-507b9dac97ff
+scopes:
+- user:info
+userUID: fe3f3a4e-b0ca-11e6-ac48-507b9dac97ff
+----
+
+[[authentication-manage-oauth-cleanup]]
+=== Cleanup Old Objects
+
+It is not possible to change an exsiting Oauth Client Authorizations.
+
+Use the following command to delete all invalid Oauth Client Authorizations:
+
+----
+oc get oauthclientauthorization -o jsonpath="{range .items[*]}{.metadata.name}{\"\\n\"}{end}" | grep -v '::' | xargs -n 1 echo oc delete oauthclientauthorization
```

Comment:
Commands that delete data are dangerous and not something I want to include in the docs.  I am ok with saying the "cluster admin has to delete...", but not ok with giving people copy paste logic that could harm their cluster.